tests/software/damageprofiler/test.yml

```diff
@@ -0,0 +1,44 @@
+- name: damageprofiler
+  command: nextflow run ./tests/software/damageprofiler -entry test_damageprofiler -c tests/config/nextflow.config -dump-channels
+  tags:
+    - damageprofiler
+  files:
+    - path: output/damageprofiler/test/3p_freq_misincorporations.txt
+      md5sum: da4cac90c78899a7cb6d72d415392b49
+    - path: output/damageprofiler/test/3pGtoA_freq.txt
+      md5sum: 8dab75d51a4b943b501d0995169c767f
+    - path: output/damageprofiler/test/5pCtoT_freq.txt
+      md5sum: fcc48ee5f72edff930d627c8bfdd8a5b
+    - path: output/damageprofiler/test/5p_freq_misincorporations.txt
+      md5sum: 54665474f5ef17dcc268567e5eaa7d86
+    - path: output/damageprofiler/test/DamagePlot_five_prime.svg
+      should_exist: true
+    - path: output/damageprofiler/test/DamagePlot.pdf
+      should_exist: true
+    - path: output/damageprofiler/test/DamagePlot_three_prime.svg
+      should_exist: true
```

Comment:
You can omit those `should_exist` statements, once you specified a path, that's basically taken care of. Doesn't hurt to leave them in, just makes the code a bit smaller.